.travis.yml

```diff
@@ -20,6 +20,12 @@ matrix:
       arch: arm64
       env: RUSTFLAGS=-Ctarget-cpu=native
 
+    # Linux (armv7)
+    - name: "armv7-unknown-linux-gnueabihf (neon)"
+      os: linux
+      arch: arm64
+      env: RUSTFLAGS=-Ctarget-feature=+neon
```

Comment:
Adding `+neon` after the fact to the `armv7-unknown-linux-gnueabihf` target results in terrible performance due to the mismatch between the pre-built `std` and what's being built with `+neon`.

The target that doesn't suffer from this problem is `thumbv7neon-unknown-linux-gnueabihf`, which bakes the `neon` option into the target definition and, therefore, comes with `neon`-enabled `std`.